runtime/etc/Makefile.mli-shared

```diff
@@ -21,7 +21,11 @@
 
 include $(CHPL_MAKE_HOME)/runtime/etc/Makefile.include
 
-CHPL_MLI_INCS = -I$(CHPL_ZMQ_LOC)/include
+ifdef CHPL_ZMQ_HOME
+CHPL_MLI_INCS = -I$(CHPL_ZMQ_HOME)/include
+CHPL_ZMQ_LIB_HOME = $(CHPL_ZMQ_HOME)/lib
+CHPL_ZMQ_DASH_L = -L$(CHPL_ZMQ_LIB_HOME)
```

Comment:
So are we sticking with the infamous DASH_L instead of SEARCH_PATH? I have no particular preference.

Reply:
See below, I think "SEARCH_PATH" is probably preferable.

Reply:
Yeah, I was looking at the Python stuff and haven't task switched back just yet.  Our discussion still stands :)